buildlib/tools/test_mad.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -exE -o pipefail
+
+IMAGE="rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/rhel8.2/builder:mofed-5.0-1.0.0.0"
+cd "$BUILD_SOURCESDIRECTORY"
```

Comment:
what about checking this var exists?

Reply:
Docker has error handling of its own.

Reply:
How is this related to Docker? this is executed every time you source this file

Reply:
What's the purpose of checking if this var exists?

Reply:
Robustness, technical debt, separation of business logic from flow management - all the things we are talking about all the time ...

Reply:
If you have a specific concern, I could address that.

Reply:
if this var is undefined the script will go to home dir and do things there